### PR TITLE
Usage metrics for 2024.3 release

### PR DESCRIPTION
**Pull request by ktuite**
Closes https://github.com/getodk/central/issues/722

Based on this PR where I change Promise.all to runSequentially for the bulk analytics queries.  https://github.com/getodk/central-backend/pull/1344 

#### Before submitting this PR, please make sure you have:

- [ ] run `make test` and confirmed all checks still pass OR confirm CircleCI build passes
- [ ] verified that any code from external sources are properly credited in comments or that everything is internally sourced

### CODE REVIEW

**Review by matthew-white — APPROVED** [review]
I haven't had time to take a close look at this and probably won't have time to today. In particular, I haven't looked at tests at all. That said, things are looking good so far. :+1: I think we should go ahead and merge, then I'll review further later. I'll leave a comment on the issue or file a new issue if I notice anything.

**Comment by matthew-white** [comment]
I've taken a closer look at everything except for tests, and things are looking good. :+1: @ktuite, how about we review tests interactively at some point?

File: test/integration/other/analytics-queries.js [hunk 1]
```diff
@@ -1079,6 +1180,59 @@ describe('analytics task queries', function () {
...
+      await asAlice.post('/v1/projects/1/datasets/people/entities')
+        .send({
+          source: { name: 'people.csv', size: 100, },
+          entities: [ { label: 'a label' }, { label: 'a label' }, { label: 'a label' } ]
+        })
+        .expect(200);
+
+      const datasets = await container.Analytics.getDatasets();
+
+      datasets[0].num_entity_create_sub_total.should.be.equal(0);
```
Comment (ktuite) [inline test/integration/other/analytics-queries.js:1228]: Run the worker and assert that this is not zero

**Review by matthew-white — COMMENTED** [review]
Just looked at tests with @ktuite, and those are looking good. :+1:

**Pull request merged**